tests/exchange/test_exchange.py

```diff
@@ -1488,6 +1488,73 @@ def test_get_valid_pair_combination(default_conf, mocker, markets):
         ex.get_valid_pair_combination("NOPAIR", "ETH")
 
 
+@pytest.mark.parametrize("base_currencies, quote_currencies, pairs_only, active_only,"
```

Comment:
@xmatthias 
Could you pls take a look at this, I'm crazy about how to make flake happy here and what it wants...